src/Sonarr.Api.V3/Queue/QueueController.cs

```diff
@@ -139,7 +139,28 @@ public object RemoveMany([FromBody] QueueBulkResource resource, [FromQuery] bool
         public PagingResource<QueueResource> GetQueue([FromQuery] PagingRequestResource paging, bool includeUnknownSeriesItems = false, bool includeSeries = false, bool includeEpisode = false, [FromQuery] int[] seriesIds = null, DownloadProtocol? protocol = null, [FromQuery] int[] languages = null, int? quality = null)
         {
             var pagingResource = new PagingResource<QueueResource>(paging);
-            var pagingSpec = pagingResource.MapToPagingSpec<QueueResource, NzbDrone.Core.Queue.Queue>(null, "timeleft", SortDirection.Ascending);
+            var pagingSpec = pagingResource.MapToPagingSpec<QueueResource, NzbDrone.Core.Queue.Queue>(
+                new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+                {
+                    "timeleft",
+                    "status",
+                    "series.sortTitle",
+                    "episode",
+                    "episodes.title",
+                    "episodes.airDateUtc",
+                    "languages",
+                    "quality",
+                    "protocol",
+                    "indexer",
+                    "downloadClient",
+                    "title",
+                    "size",
+                    "estimatedCompletionTime",
+                    "added",
+                    "progress"
```

Comment:
If requiring them makes the most sense to fix this then I think they should be sorted, to make it easier to read.

```suggestion
                    "added",
                    "downloadClient",
                    "episode",
                    "episodes.airDateUtc",
                    "episodes.title",
                    "estimatedCompletionTime",
                    "indexer",
                    "languages",
                    "progress",
                    "protocol",
                    "quality",
                    "series.sortTitle",
                    "size",
                    "status",
                    "timeleft",
                    "title"
```